<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fırat Üniversitesi Anda Jet</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>WhatsApp-Image-2022-05-18-at-12.38.31.ico</normaloff>WhatsApp-Image-2022-05-18-at-12.38.31.ico</iconset>
  </property>
  <widget class="QPushButton" name="btnYapisal">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>170</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Yapısal Hesaplamalar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnItki">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>220</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Eksenel İtki Hesabı</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>İSTİKBAL GÖKLERDEDİR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
